turbinia/workers/__init__.py

```diff
@@ -279,6 +281,11 @@ def execute(
     result.error['stderr'] = stderr
     ret = proc.returncode
 
+    for file_ in log_files:
+      result.log('Output file at {0:s}'.format(file_))
```

Comment:
A small thing, but could you maybe add a quick check here to make sure that the file actually exists before we try to save it?  I think there is a check further down the stack here, but it would be nice to have a relevant error message here.

Reply:
I'll just merge as is and add a check after.  Thanks!

Reply:
Okay, sorry about not getting that done. I've been sick for the past week and a half and haven't felt like doing much of anything.